wd_demo.au3

```diff
@@ -5,18 +5,19 @@
 #include <ButtonConstants.au3>
 #include <WindowsConstants.au3>
 
-Local Const $sElementSelector = "//input[@name='q']"
+#Region - Global's declarations
 
-Local $sDesiredCapabilities, $iIndex, $sSession
-Local $nMsg, $bProcess = False
-
-Local $aBrowsers[][2] = _
-		[["Firefox", SetupGecko], _
+Global Const $sElementSelector = "//input[@name='q']"
+Global Const $aBrowsers[][2] = _
```

Comment:
These globals could be moved to locals of new _WD_Demo function

Reply:
checking

Reply:
$sElementSelector is used in `DemoElements()` and `DemoActions()` so I would'nt want to move them.
What is your opionion ?

$aBrowsers is used only in `_WD_Demo()` but I want to split them to 2 spearate function and do not exit main look in another way then using $GUI_EVENT_CLOSE.

Reply:
Right... I was referring to the globals below $sElementSelector. 

> $aBrowsers is used only in _WD_Demo() but I want to split them to 2 spearate function and do not exit main loop in another way then using $GUI_EVENT_CLOSE.

Not sure what you mean here. Please explain.

Reply:
For this moment I porpose that you accept this PR as it is.
Then I do my next PR which will show to you exactly what I mean.